cmake/FDBComponents.cmake

```diff
@@ -51,36 +51,58 @@ endif()
 # Java Bindings
 ################################################################################
 
-set(WITH_JAVA OFF)
-find_package(JNI 1.8)
-find_package(Java 1.8 COMPONENTS Development)
-# leave FreeBSD JVM compat for later
-if(JNI_FOUND AND Java_FOUND AND Java_Development_FOUND AND NOT (CMAKE_SYSTEM_NAME STREQUAL "FreeBSD"))
-  set(WITH_JAVA ON)
-  include(UseJava)
-  enable_language(Java)
+option(BUILD_JAVA_BINDING "build java binding" ON)
+if(NOT BUILD_JAVA_BINDING)
+  set(WITH_JAVA OFF)
 else()
   set(WITH_JAVA OFF)
+  find_package(JNI 1.8)
+  find_package(Java 1.8 COMPONENTS Development)
+  # leave FreeBSD JVM compat for later
+  if(JNI_FOUND AND Java_FOUND AND Java_Development_FOUND AND NOT (CMAKE_SYSTEM_NAME STREQUAL "FreeBSD"))
+    set(WITH_JAVA ON)
+    include(UseJava)
+    enable_language(Java)
+  else()
+    set(WITH_JAVA OFF)
+  endif()
 endif()
 
 ################################################################################
 # Python Bindings
 ################################################################################
 
-find_package(Python COMPONENTS Interpreter)
-if(Python_Interpreter_FOUND)
-  set(WITH_PYTHON ON)
-else()
-  #message(FATAL_ERROR "Could not found a suitable python interpreter")
+option(BUILD_PYTHON_BINDING "build python binding" ON)
+if(NOT BUILD_PYTHON_BINDING)
```

Comment:
Multiple things in the build system (at least the C bindings and the documentation) depend on python. So I think you need to run `find_package(Python COMPONENTS Interpreter)` even if you don't want to build the python bindings.

Reply:
Thanks for the tips :+1: